gcp/extract/quantiles.py

```diff
@@ -42,6 +42,14 @@
 
 for filestuff in data:
     print "Creating file: " + str(filestuff)
+    
+    if configs.is_parallel_deltamethod(config):
+        try:
+            output_check = parallel_deltamethod_data[filestuff].keys()
+        except KeyError:
+            print str(filestuff) + " is not in delta method output. Skipping model specification..."
+            continue
```

Comment:
Can we replace this with:
```
if configs.is_parallel_deltamethod(config) and filestuff not in parallel_deltamethod_data:
    print str(filestuff) + " is not in delta method output. Skipping model specification..."
    continue
```
I think that is somewhat more legible.

Reply:
yep, much prettier :). ill test it out now.